components/compliance-service/compliance.go

```diff
@@ -177,6 +178,10 @@ func serveGrpc(ctx context.Context, db *pgdb.DB, connFactory *secureconn.Factory
 		if err != nil {
 			logrus.WithError(err).Fatal("could not register project update task executors")
 		}
+		err = project_purge.RegisterTaskExecutors(cerealProjectUpdateManager, "compliance", ingesticESClient)
+		if err != nil {
+			logrus.WithError(err).Fatal("could not register project purge task executors")
+		}
```

Comment:
Each of the three services adds this or equivalent to receive the purge directions.